src/App.tsx

```diff
@@ -1,35 +1,19 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import './App.css';
+import { Checkbox } from './components/Checkbox/Checkbox.tsx';
+import { CheckboxSize, CheckboxColor } from './components/Checkbox/Checkbox.ts';
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <Checkbox
```

Comment:
Pro účely testování by asi bylo lepší vytvořit komponentu `Example.tsx`, tu importovat zde, a až do ní přidat Checkbox.
Aby sis správně přidala Checkbox s react-hook-form, tak potřebuješ zde přidat ještě `useForm` a zachytávat callback. Zbytečně bychom plevelily `App.tsx`.